lib/click.ts

```diff
@@ -1,8 +1,12 @@
-import { InsightsEvent } from "./_sendEvent";
+export interface InsightsSearchClickEvent {
+  eventName: string;
+  userID: string;
+  timestamp: number;
+  indexName: string;
 
-export interface InsightsSearchClickEvent extends InsightsEvent {
   objectID: string | number;
   position: number;
+
   queryID?: string;
```

Comment:
From what I understand inside the code the `queryID` is required.

Reply:
will fix that